io.github.shiiion.primehack.yml

```diff
@@ -0,0 +1,132 @@
+app-id: io.github.shiiion.primehack
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: dolphin-emu-wrapper
+rename-desktop-file: dolphin-emu.desktop
+rename-icon: dolphin-emu
+rename-appdata-file: dolphin-emu.appdata.xml
+finish-args:
+  - --device=all
+  # the file picker uses portals but the set
+  # game directory feature still needs this
+  - --filesystem=host:ro
```

Comment:
`host`? that's very broad.

(and the Dolphin does it out is not a proper justification)

Reply:
To elaborate further on the existing comment, this is so that users can select their game or ISO folder, which not only is used to view and start their games, but to edit game specific settings and propreties. A user might wish to place their games folder outside of their home directory or perhaps on a removable storage device.

If it is a 'flatpak sin' to be doing this and there's a particular reason it should be restricted, regardless of the read-only state, then we can compromise by restricting it to the home folder, though this won't be consistent with the existing vanilla package and may require the user to adjust their settings accordingly.